macros/tableau.pl

```diff
@@ -356,27 +356,22 @@ sub phase1_solve {
 	[complementary_basis_set] = $self->primal_basis_to_dual(primal_basis_set)
 	[complementary_basis_set] = $self->dual_basis_to_primal(dual_basis_set)
 
```

Comment:
I think there needs to be a "=cut" line here.

Reply:
yep -- thanks.